base/MAPL_EsmfRegridder.F90

```diff
@@ -1468,6 +1468,7 @@ subroutine create_route_handle(this, kind, rc)
            rh_file_exists = .false.
         end if
         if (rh_file_exists) then
+           if (mapl_am_I_root()) write(*,*)"bmaa reading "//trim(rh_file)
```

Comment:
If we want this print here, it should be a logger call...

Reply:
We don't have access to the logger here.

Reply:
Hmm. Well, if we don't have access to logger, then if nothing else the print should be "generalized". No initials, etc. but make it look like a "MAPL" print.

Reply:
I changed it so it's just the filename and the message

Reply:
Ah. Okay. I guess I thought github would update this comment thing. I guess not!

Reply:
And certainly without anyone's initials.